students/eoner/lesson02/grid.py

```diff
@@ -0,0 +1,6 @@
+def print_grid(x, s):
+    for i in range(x):
+        print((((("+" + ("-"*s)))*x) + "+")+(("\n"+((("|" + (" "*s)))*x) + "|")*s))
+    print(((("+" + ("-"*s)))*x) + "+")
+
+print_grid(3,4) 
```

Comment:
You could include some comments, doc strings, etc. but this definitely does work. Thanks.